README.md

```diff
@@ -31,20 +31,8 @@ HA requires the following:
 * A blob store location for storing components that can be commonly accessed by all active nodes
 * Connectivity between Nexus Repository, the database, and blob store
 * All architecture must be in one region (if deploying to cloud) or data center (if deploying on-premises)
+* A physical disk for storing Nexus Repository logs and configuration settings used for generating the support zip
```

Comment:
* We should simply say `disk` here as it could be a network attached storage.
* Support zips should only need disk storage if there is no blobstore

Reply:
Done